pkg/metamorph/client.go

```diff
@@ -181,6 +211,11 @@ func (m *Metamorph) Health(ctx context.Context) error {
 
 // SubmitTransaction submits a transaction to the bitcoin network and returns the transaction in raw format.
 func (m *Metamorph) SubmitTransaction(ctx context.Context, tx *sdkTx.Transaction, options *TransactionOptions) (*TransactionStatus, error) {
+	if m.tracer != nil {
+		_, span := m.tracer.Start(ctx, "SubmitTransaction")
```

Comment:
I think here the context form tracer has to be returned so that it is used in line 242, not?

```go
response, err = m.client.PutTransaction(ctx, request)
```